perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -241,11 +255,68 @@ def SetFiles(self):
       self.RemoteCommand('echo "%s" | sudo tee %s' %
                          (value, path))
 
+  def ApplySysctlPersistent(self, key, value):
+    """Apply "key=value" pair to /etc/sysctl.conf and reboot.
+
+    The reboot ensures the values take effect and remain persistent across
+    future reboots.
+
+    Args:
+      key: a string - the key to write as part of the pair
+      value: a string - the value to write as part of the pair
+    """
+
```

Comment:
getting picky here, but there shouldn't be a space after docstring. see https://www.python.org/dev/peps/pep-0257/

Reply:
by space i mean blank line